server.R

```diff
@@ -124,4 +124,33 @@ shinyServer(function(input, output) {
             ungroup()
         create_benthic_comp_plot(data_filtered, input, benthic_comp_plot_caption())
     })
+
+    output$keyCollectors <- renderInfoBox({
+        infoBox(
+            "Data Collectors",
+            collectors_count,
+            icon = shiny::icon(NULL)
+        )
+    })
+    output$keyYears <- renderInfoBox({
+        infoBox(
+            "Years",
+            years_count,
+            icon = shiny::icon(NULL)
+        )
+    })
+    output$keyLocale <- renderInfoBox({
+        infoBox(
+            "Localities",
+            localities_count,
+            icon = shiny::icon(NULL)
+        )
+    })
+    output$keySites <- renderInfoBox({
+        infoBox(
+            "Sites",
+            sites_count,
+            icon = shiny::icon(NULL)
+        )
+    })
```

Comment:
Could you swap the order of the info boxes? Where the title is the number and the value is the label?
For example:
36 
Data Collectors

Instead of:
Data Collectors
36